sos/report/plugins/openstack_glance.py

```diff
@@ -70,7 +70,18 @@ def setup(self):
                                     "intended to connect to the OpenStack "
                                     "environment.")
             else:
-                self.add_cmd_output("openstack image list --long")
+                self.add_cmd_output(
+                    "openstack image list --long --all-projects"
+                )
```

Comment:
We are calling very similar command in `exec_cmd` few lines below. Can't we get the list of images from this output, if we would call here `self.collect_cmd_output("openstack image list --long --all-projects")`? Or is parsing that output worse than calling an extra similar command once again, like you do now?

Reply:
let me see what I can do, the less commands we run the better for performance. I was in 2 minds before as well. But, now that you've questioned it as well, I'll try re-factoring it, and see where I can get this to